lib/assessment.md

```diff
@@ -161,7 +174,4 @@ Consider the following media queries. How wide would the screen need to be for e
   }
 }
 ```
-
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+Screen needs to be 1200px
```

Comment:
Nope, check your math. At 1200px width, both media queries are 'visible', and since CSS cascades, the last rule is the one that wins out. 30% of 1200px is 360px, not 150px.